### PR TITLE
Cleanup ./scripts/run.sh

### PR DESCRIPTION
**Pull request by patrick-ogrady**
image

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -132,6 +179,7 @@ while [[ ! -s /tmp/avalanchego-v${VERSION}/output.yaml ]]
 done
 
 if [[ -f "/tmp/avalanchego-v${VERSION}/output.yaml" ]]; then
+  # TODO: ADD /RPC and /WS ENDING TO OUTPUT.YAML
```
Comment (patrick-ogrady) [inline scripts/run.sh:182]: Need to modify YAML in place

Comment (patrick-ogrady) [reply scripts/run.sh:182]: example:
```yaml
endpoint: /ext/bc/2Z36RnQuk1hvsnFeGWzfZUfXNr7w1SjzmDQ78YxfTVNAkDq3nZ
logsDir: /var/folders/mp/6jm81gc11dv3xtcwxmrd8mcr0000gn/T/runnerlogs2984620995
pid: 55547
uris:
- http://localhost:61278
- http://localhost:61280
- http://localhost:61282
- http://localhost:61284
- http://localhost:61286
```

**Comment by patrick-ogrady** [comment]
TODO: accept arg for address with allocations

**Review by darioush — APPROVED** [review]

**Review by gyuho — APPROVED** [review]

**Review by aaronbuchwald — COMMENTED** [review]
LGTM

**Pull request merged**